docs/Contributing/Testing-and-local-development.md

```diff
@@ -607,6 +607,60 @@ To use the workflow, follow these steps:
   - `fleet-base.pkg`
 4. Download them to your workstation.
 
+#### Building a signed fleetd-base installer from `local TUF`and signing with Apple Developer Account
+
+1. Build fleetd from your [local TUF](https://github.com/fleetdm/fleet/blob/HEAD/docs/Contributing/Run-Locally-Built-Fleetd.md) service
```

Comment:
Should mention this is the "base" package so `fleetctl package` needs to be executed with `--use-system-configuration` and without `--fleet-url` or `--enroll-secret` (e.g. [here](https://github.com/fleetdm/fleet/blob/729fdd2c56c9d51b5b8f95f37ed7f0f67c3c8d23/.github/workflows/build-fleetd-base-pkg.yml#L73-L77))

Reply:
```suggestion
1. Build fleetd base pkg installer from your [local TUF](https://github.com/fleetdm/fleet/blob/HEAD/docs/Contributing/Run-Locally-Built-Fleetd.md) service by running the following command after the local TUF repository is generated `fleetctl package --type=pkg --update-roots=$(fleetctl updates roots --path ./test_tuf) --disable-open-folder --debug --update-url=$LOCAL_TUF_URL --enable-scripts --use-system-configuration`.
```